tripy/tests/frontend/module/test_module.py

```diff
@@ -304,3 +304,77 @@ def test_state_dict(self, complex_network):
         assert module.nets["dict_net"].dummy_dict["op1"].nested.param is tensor
         assert module.nets["list_net"].dummy_list[0].nested.param is tensor
         assert module.nets["list_net"].dummy_list[1].nested.param is tensor
+
+
+class TestMixedContainerNetwork:
+    def test_basic_structure(self, mixed_container_network):
+        assert hasattr(mixed_container_network, "mixed_list")
+        assert hasattr(mixed_container_network, "mixed_dict")
+
+        assert isinstance(mixed_container_network.mixed_list, list)
+        assert isinstance(mixed_container_network.mixed_dict, dict)
+
+        assert any(isinstance(item, tp.Module) for item in mixed_container_network.mixed_list)
+        assert any(callable(item) and not isinstance(item, tp.Module) for item in mixed_container_network.mixed_list)
+
+        assert any(isinstance(value, tp.Module) for value in mixed_container_network.mixed_dict.values())
+        assert any(
+            callable(value) and not isinstance(value, tp.Module)
+            for value in mixed_container_network.mixed_dict.values()
+        )
+
+    def test_named_children(self, mixed_container_network):
+        children = list(mixed_container_network.named_children())
+        for _, child in children:
+            assert isinstance(child, tp.Module)
```

Comment:
This isn't testing that we see all the children we expect to see. Maybe we can also check the count?

Reply:
added count check